src/MovieCard/MovieCard.js

```diff
@@ -105,11 +94,10 @@ class MovieCard extends Component {
         ☆
       </button>
     );
-    return favoriteMovieIDs.includes(this.props.id) ? favorited : notFavorited
+    return favoriteMovieIDs.includes(this.props.id) ? favorited : notFavorited;
   } 
-  
   createRadioButtons = () => {
-    let radioButtons = [];
+    const radioButtons = [];
```

Comment:
so we can use const here because we are just pushing into an array and not reassigning?